app/clients/salesforce/salesforce_auth.py

```diff
@@ -1,7 +1,16 @@
+import requests
 from flask import current_app
 from simple_salesforce import Salesforce
 
 
+class TimeoutAdapter(requests.adapters.HTTPAdapter):
+    """Custom adapter to add a timeout to Salesforce API requests"""
+
+    def send(self, *args, **kwargs):
+        kwargs["timeout"] = 10
```

Comment:
Let's make this an all caps constant defined at the top, something like `SALESFORCE_TIMEOUT_SECONDS`

Reply:
Good idea - just added